README.md

```diff
@@ -1,190 +1,101 @@
+         files: 
+           path:
+             - path: {path}/ldbc_snb_datagen/social_network/dynamic/person.csv
+             - path: {path}/ldbc_snb_datagen/social_network/static/place.csv
+             - path: {path}/ldbc_snb_datagen/social_network/static/organisation.csv
+             - path: {path}/ldbc_snb_datagen/social_network/dynamic/post.csv
+             - path: {path}/ldbc_snb_datagen/social_network/dynamic/comment.csv
+             - path: {path}/ldbc_snb_datagen/social_network/dynamic/forum.csv
+             - path: {path}/ldbc_snb_datagen/social_network/static/tag.csv
+             - path: {path}/ldbc_snb_datagen/social_network/static/tagclass.csv
+             ...
+           
+###     Import ldbc data to ldbc:
+        ./nebula-importer --config  ldbc_vid_int.yaml
+ 
+#  4、Perf test
+##     Mvn package
+       cd util/LdbcGoStep/ && mvn package
+      
+      
+##     Put  jar and jmx file to Jmeter 
+       LdbcGoStep-2-jar-with-dependencies.jar: put to  apache-jmeter-5.4/lib/ext
+       nebula-bench/ldbc/jmx/go_step.jmx : put to apache-jmeter-5.4/
+       config jmx: 
+         <stringProp name="LoopController.loops">{loops}</stringProp> 
+         <stringProp name="ThreadGroup.num_threads">{nums}</stringProp> 
+        
+         <stringProp name="Argument.name">hosts</stringProp>
+         <stringProp name="Argument.value">{ip1:port,ip2:port,ip3:port}</stringProp>
+         
+         <stringProp name="Argument.name">maxconn</stringProp>
+         <stringProp name="Argument.value">{max}</stringProp>  //  >= 1
+        
+         <stringProp name="Argument.name">user</stringProp>
+         <stringProp name="Argument.value">{user}</stringProp>
+              
+         <stringProp name="Argument.name">pwd</stringProp>
+         <stringProp name="Argument.value">{pwd}</stringProp>
+               
+         <stringProp name="Argument.name">space</stringProp>
+         <stringProp name="Argument.value">{spacename}</stringProp>
+         
+         <stringProp name="Argument.name">nGQL</stringProp>
+         <stringProp name="Argument.value">{GO 3 STEP FROM "replace" OVER knows}</stringProp>
+    
+##     Nebula perf test nGQL
+       GO 1 STEP FROM replace OVER knows 
+       GO 1 STEP FROM replace OVER knows YIELD knows.`time`, $$.person.first_name, $$.person.last_name, $$.person.birthday
+       GO 2 STEP FROM replace OVER knows 
+       GO 2 STEP FROM replace OVER knows YIELD knows.`time`, $$.person.first_name, $$.person.last_name, $$.person.birthday
+       GO 3 STEP FROM replace OVER knows 
+       GO 3 STEP FROM replace OVER knows YIELD knows.`time`, $$.person.first_name, $$.person.last_name, $$.person.birthday
+
+
+##    Run Jmeter
+       cd apache-jmeter-5.4 
+       perf test:
+        ./bin/jmeter.sh -n -t go_step.jmx  -l go_step.jtl -j go_step.log
+       report(jmeter):
+        ./bin/jmeter.sh -g go_step.jtl  -o go_step
+       report(statictis):
+        python3 ldbc/scripts/statistics.py go_step.jtl     

```

Comment:
Would you please merge this part into a script? That would be helpful much more to the users.